package.json

```diff
@@ -71,8 +72,9 @@
     "jest": "^24.5.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^1.14.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "check-prop-types": "1.1.2",
+    "react": "16.8.6",
+    "react-dom": "16.8.6",
```

Comment:
Probably safer :relieved:. But, also requires we be more aware of when `react` and `react-dom` updates and respond accordingly.

Reply:
I think it's one last safe version before react 17, tried 16.9.0, it gives you a lot of deprecation warnings